docs/quick-release-notes/main.go

```diff
@@ -0,0 +1,167 @@
+	} `json:"commit"`
+	NodeID string `json:"node_id"`
+}
+
+func getTagsFromAPI(owner string, repo string) []Tag {
+	// https://docs.github.com/en/rest/reference/repos#list-repository-tags
+	pathToTags := fmt.Sprintf("https://api.github.com/repos/%v/%v/tags", owner, repo)
+	resp, err := http.Get(pathToTags)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	sb := string(body)
+	var tagsForEachRelease []Tag
+	json.Unmarshal([]byte(sb), &tagsForEachRelease)
+	return tagsForEachRelease
+}
+
+func getChangeLogInfo(scanner *bufio.Scanner) ([]int, []string, int, []string) {
+	// regexp for: ## 1.0.0 (2021-12-13)
+	r, err := regexp.Compile(`## \d+\.\d+\.\d+`)
+	if (err) != nil {
+		log.Fatal(err)
+	}
+	lineCount := 0
+	counterForChangeLogLength := 0
+	var versionLineNumbers []int
+	var changeLogText []string
+	var versionTitlesForEachRelease []string
+	for scanner.Scan() {
+		counterForChangeLogLength++
+		changeLogText = append(changeLogText, scanner.Text())
+		if r.MatchString(scanner.Text()) {
+			// ## 2.3.0 (2022-02-24) => 2.3.0 (2022-02-24)
+			versionNumberAndDateWithoutHashes := scanner.Text()[3:]
+			versionTitlesForEachRelease = append(versionTitlesForEachRelease, versionNumberAndDateWithoutHashes)
+			versionLineNumbers = append(versionLineNumbers, lineCount)
+		}
+		lineCount++
+	}
+	if scanner.Err() != nil {
+		fmt.Println(scanner.Err())
+	}
+	return versionLineNumbers, changeLogText, counterForChangeLogLength, versionTitlesForEachRelease
+}
+
+func makeReleaseSlice(versionLineNumbers []int, changeLogText []string, changeLogLength int) []string {
+	// Store contributors and errors from end of changelog.
+	contributorsAndErrors := ""
+	numberOfLinesForLastRelease := 5
+	for i := numberOfLinesForLastRelease + versionLineNumbers[len(versionLineNumbers)-1]; i < changeLogLength; i++ {
+		contributorsAndErrors += changeLogText[i] + "\n"
+	}
+	var releaseNotesForEachVersion []string
+	// exclude Last version (## 0.2.0) with - 1
+	versionLineNumbersLengthMinusOne := len(versionLineNumbers) - 1
+	for i, versionLineNumber := range versionLineNumbers[:] {
+		releaseBodyLines := ""
+		if i < (versionLineNumbersLengthMinusOne) {
+			for j := versionLineNumbers[i] + 1; j < versionLineNumbers[i+1]; j++ {
+				releaseBodyLines += changeLogText[j] + "\n"
+			}
+		}
+		// Handle last version (## 0.2.0)
+		if versionLineNumber == versionLineNumbers[versionLineNumbersLengthMinusOne] {
+			for j := 1; j < numberOfLinesForLastRelease; j++ {
+				releaseBodyLines += changeLogText[versionLineNumber+j] + "\n"
+			}
+		}
+		releaseNotesForEachVersion = append(releaseNotesForEachVersion, releaseBodyLines+contributorsAndErrors)
+	}
+	return releaseNotesForEachVersion
+}
+
+func sendToGitHubAPI(tagForRelease Tag, releaseNote string, versionTitle string, owner string, repo string) {
+	// https://docs.github.com/en/rest/reference/releases
+	postBody, _ := json.Marshal(map[string]string{
```

Comment:
What is `_`? If it's an error, check for it!

Reply:
Yeah it is an error slightly confusing that Unmarshal syntax is:   `error = json.Unmarshal()` and Marshal is `variable, error := json.Marshal()`... :mag: 
`